.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If prettier is not a dependency, you don't need to provide the config to use it. If you are using locally thats fine, but this file should be added to your gitignore if so